client/src/pages/Filtering.js

```diff
@@ -0,0 +1,54 @@
+import axios from "axios";
+import { useQuery } from "react-query";
+import { useState } from "react";
+import Layout from "../components/Layout";
+import IconButton from "@mui/material/IconButton";
+import SearchIcon from "@mui/icons-material/Search";
+import "../index.css";
+
+//Search Component- Read in CRUD
+
+function App() {
+  const [search, setNewSearch] = useState("");
+  const { isLoading, data, error } = useQuery(["posts"], () =>
+    axios("https://demo5940257.mockable.io/menu_items")
+  );
+  if (error) return <h2>Error </h2>;
+  if (isLoading) return <h2> isLoading </h2>;
+  console.log(data);
+
+  const handleSetSearch = (e) => {
+    setNewSearch(e.target.value);
+  };
+
+  const filtered = !search
+    ? data.data
+    : data.data.filter((data) =>
+        data.title.toLowerCase().includes(search.toLowerCase())
+      );
```

Comment:
GREAT work on this!  A minor enhancement here would be to make the variable name more specific.  instead of `filtered` you could go with `filteredMenuItems` or something similar.  Anything to help future you remember what that line of code is doing.

Reply:
I will merge in and then rename - I'm working at better naming conventions